src/ConcertDisplay.js

```diff
@@ -6,15 +6,29 @@ export default class ConcertDisplay extends Component {
   constructor() {
     super();
   }
-
+  
   render() {
-    const {concertData, setlistData} = this.props;
-    return (
-      <main className="concert-display">
-      {concertData.map(concert => {
-        return <Concert concert={concert}/>
-      })}
-      </main>
-    )
+    const {concertData, setlistData, currentSearch} = this.props;
+    if(currentSearch === null) {
+      return (
+        <main className="concert-display">
+        {this.props.concertData.map(concert => {
+          return <Concert concert={concert}/>
+          })}
+        </main>
+      )
+    } else {
+      return (
+        <main className="concert-display">
+        {this.props.concertData.filter(concert => {
+          return concert.venue.name.toLowerCase().includes(currentSearch) || concert.date.includes(currentSearch) || concert.venue.location.toLowerCase().includes(currentSearch)
```

Comment:
Also still need to add the song searching, which we talked about earlier. This looks great for the venues and dates though!

Reply:
I'll add an issue for that so we don't forget